src/main/java/com/meow/Meowception.java

```diff
@@ -0,0 +1,35 @@
+package com.meow;
+public class Meowception extends Exception {
+    String code;
+    
+    // public consturctor for Meowception
```

Comment:
Is this a typo? Should there be more explanation for the constructor to be clearer?